### ISSUE TITLE
command not found: ara

### ISSUE THREAD

**Issue by hawksight** [ansible-community/ara#172]
## What is the issue ?

Following the docs [here](https://ara.readthedocs.io/en/latest/getting-started.html#recording-playbooks-with-an-api-server) I have the api-server container running.

When i install `ara` package inside my virtual environment the command is not found:

```
(.python) 
[peter@envy] ~/projects/sopost/gcloud-infrastructure/ansible (stg-eu/db-perms) ⚡ 
❯ ara playbook list

zsh: command not found: ara

```

Package is installed but there is no CLI command installed to run: `ara playbook list`.

```
❯ pip3 list | head -10

Package             Version
------------------- ---------
ansible             2.9.13
ansible-lint        4.3.4
apache-libcloud     3.1.0
ara                 1.4.3
arrow               0.16.0
attrs               20.2.0
backcall            0.2.0
```

Note that I cannot run the exact command in the documentation because I am using a virtual env:

```
❯ python3 -m pip install --user ansible ara                                                            ⏎

ERROR: Can not perform a '--user' install. User site-packages are not visible in this virtualenv.
                                                                                                  
```

Versions:

```
python3 - 3.8.2
Ubuntu - 20.04
pip 20.2.3
ansible 2.9.13
```

## What should be happening ?

Command should be found if `ara` in installed via pip.

I have tried 1.4.2 and 1.4.3 to see if it was a recent version issue.

Unless i've missed something and I can't install inside a vritual env?

**Comment by hawksight** [comment]
Might it be there is no python 3.8 in the setup.cfg?

https://github.com/ansible-community/ara/blob/1.4.3/setup.cfg#L17-L21

**Comment by dmsimard** [comment]
Hey @hawksight and thanks for the issue.

The ``ara`` command hasn't been released yet but it is merged in master which is why it's in the latest docs. Sorry about that.
It'll be released in v1.5 which is due sometime soon.

If you'd like to try out the CLI in the meantime (very much looking for feedback!), you can either install the [first beta release for 1.5](https://github.com/ansible-community/ara/releases/tag/1.5.0b1) (``python3 -m pip install --user --pre ara[server]``) or install from source using the master branch.

**Comment by dmsimard** [comment]
o/ @hawksight 

1.5.0 is planned to be released next week so you won't need to use ``--pre`` for using the CLI when it's out. The full changelog is available [here](https://github.com/ansible-community/ara/releases/tag/1.5.0rc1).

I'll close this issue, feel free to open a new one if there's anything else.
